### PR TITLE
fix for #1812

### PR DESCRIPTION
**Pull request by atanasster**
What does this PR do?

if the drop control is hidden, find first visible parent

#1812

### CODE REVIEW

File: src/js/utils/DOM.js [hunk 1]
```diff
@@ -110,14 +110,24 @@ export const makeNodeUnfocusable = (node) => {
   }
 };
 
+export const findVisibleParent = (element) => {
+  if (element) {
+    return element.offsetParent ?
+      element : findVisibleParent(element.parentElement);
+  }
+  return null;
```
Comment (alansouzati) [inline src/js/utils/DOM.js]: can we use undefined instead of null?

File: src/js/utils/DOM.js [hunk 2]
```diff
@@ -110,14 +110,24 @@ export const makeNodeUnfocusable = (node) => {
...
   filterByFocusable,
   findScrollParents,
+  findVisibleParent,
   makeNodeFocusable,
   makeNodeUnfocusable,
   getBodyChildElements,
   getNewContainer,
   setTabIndex,
   unsetTabIndex,
+
```
Comment (alansouzati) [inline src/js/utils/DOM.js]: i think we can remove this extra line here

File: src/js/components/Drop/DropContainer.js [hunk 3]
```diff
@@ -73,7 +73,7 @@ class DropContainer extends Component {
       container.style.maxHeight = '';
 
       // get bounds
-      const controlRect = control.getBoundingClientRect();
+      const controlRect = findVisibleParent(control).getBoundingClientRect();
```
Comment (alansouzati) [inline src/js/components/Drop/DropContainer.js]: find visible parent might return undefined, so `getBoundingClientRect` may fail to execute. I believe this is the reason the build is failing

**Comment by atanasster** [comment]
fix for failed tests

**Comment by atanasster** [comment]
Merged #1832.

**Pull request closed without merging**